homeassistant/components/nfandroidtv/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow for NFAndroidTV integration."""
+from __future__ import annotations
+
+import logging
+
+from notifications_android_tv.notifications import ConnectError, Notifications
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class NFAndroidTVFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for NFAndroidTV."""
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        errors = {}
+
+        if user_input is not None:
+            host = user_input[CONF_HOST]
+            name = user_input[CONF_NAME]
+
+            await self.async_set_unique_id(host)
```

Comment:
We normally don't allow using host as unique id. What kind of value is the host?

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-id-requirements